R/tokens.R

```diff
@@ -208,13 +268,14 @@ as.tokens <- function(x, concatenator = '_') {
 #' @export
 as.tokens.list <- function(x, concatenator = '_') {
     result <- structure(tokens_hash(x),
-                        "class" = c("tokens", "tokenizedTexts"),
-                        "what" = "word",
-                        "ngrams" = 1L,
-                        "skip" = 0L, 
-                        "concatenator" = concatenator,
-                        "padding" = FALSE)
-    docvars(result) <- data.frame(row.names = docnames(result))
```

Comment:
Wouldn't all this be easier if we had an S4 class for this with a prototype to set these slots?